src/editor/EditorManager.js

```diff
@@ -91,7 +91,7 @@ define(function (require, exports, module) {
      * @private
      */
     var _inlineDocsProviders = [];
-
+    var _inlineEditProviderCheck = [];
```

Comment:
_inlineEditProviders[] uses the htmlToCSSProvider function in CSSInlineEditor.js that is set by the function registerInlineEditProvider, htmlToCssProvider currently checks and does everything that we need but also creates the inline viewer that pops up. Since we only need to check the current editor and pos, i registered _inlineEditProviderCheck[] to use the new function i created in CSSinelineEditor.